wpilibc/src/main/native/cpp/ADIS16448_IMU.cpp

```diff
@@ -509,6 +510,11 @@ void ADIS16448_IMU::Reset() {
   m_integ_gyro_angle_z = 0.0;
 }
 
+void ADIS16448_IMU::Reset(Rotation3d angle) {
```

Comment:
Would it also be worth to add a Rotation 2d version of the overload with the Z axis set to 0?

Reply:
Rotation3d explicitly didn't support construction from Rotation2d because you can make a 2D slice along any axis in principle.